cmd/clairctl/docker/pull.go

```diff
@@ -0,0 +1,88 @@
+package docker
```

Comment:
Same here, could we somehow use Docker as a library to do that instead of manually fetching the Docker API? I'm not sure I understand the architecture / idea though. Eventually, we'll want to make clairctl work with rkt images too.